src/geo/gmaps/gmaps-map-view.js

```diff
@@ -65,6 +66,10 @@ var GoogleMapsMapView = MapView.extend({
       self.trigger('dblclick', e);
     });
 
+    google.maps.event.addListener(this._gmapsMap, 'bounds_changed', function (e) {
```

Comment:
What about adding a test for google and leaflet when "bounds" has changed and the function is triggered?